KILTprotocol/sdk-js
# Dn api 163 public identity methods

## fixes #201 

Reverted changes to have the examples of the method be more extensive

## How to test:


If you'd like to see how the doc looks like:
Run npm run build:docs. This generates the doc at sdk-js/docs/api.
Open the so-generated files in your browser: sdk-js/docs/api/classes/attestation.attestation-1.html.

## Checklist:
